python/ray/dag/BUILD

```diff
@@ -72,8 +72,9 @@ py_test(
 
 py_test_module_list(
   files = [
-    "tests/experimental/test_accelerated_dag.py",
-    "tests/experimental/test_torch_tensor_dag.py",
+    # Commented out tests are flaky and not ready to be run in CI yet.
+    # "tests/experimental/test_accelerated_dag.py", 
+    # "tests/experimental/test_torch_tensor_dag.py",
```

Comment:
fwiw, you can just add `manual` in the `tags`, and then it won't be selected by `bazel query`, but will still be runnable with `bazel test` when explicitly being asked to.